File: tests/fixtures/local-dev-samples-legacy/node-hello/handler.js
module.exports.handler = async (event, context) => {
  return {
    status: 200,
    headers: {
      "Content-Type": "application/json"
    },
    body: JSON.stringify({
      message: "Hello from Node!",
      runtime: "node"
    })
  };
};

File: tests/fixtures/dep-isolation/rust/rust-basic/handler.rs
use serde_json::{json, Value};

pub fn handler(_event: Value) -> Value {
    json!({
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json!({
            "ok": true,
            "runtime": "rust",
            "message": "Rust function with serde_json only"
        }).to_string()
    })
}
